libs/openFrameworks/gl/ofVboMesh.cpp

```diff
@@ -168,11 +168,10 @@ void ofVboMesh::updateVbo(){
 			if(getNumColors()==0){
 				vbo.clearColors();
 				vboNumColors = getNumColors();
-			}else if(vboNumColors<getNumColors()){
```

Comment:
Having less colors than vertices could lead to undefined graphics behaviour. I believe you should not be able to remove colors from a vbo - just updating them once the vbo has been uploaded to VRAM. 

Also, we need to check that the number of colors is in sync with the number of vertices. I can't really see a use-case anyway where you would want to "remove" vertex colors. A (vbo) mesh either has a color attribute for the whole mesh, or it hasn't. If it has a color attribute that's for the whole mesh.

I think the remove color method may be misleading, and we're proably better off changing that method than trying to guess a programmer's intention so far down the (pipe)line ...